oauth2client/_helpers.py

```diff
@@ -26,11 +26,6 @@
 from six.moves import urllib
 
 
-__author__ = (
-    'rafek@google.com (Rafe Kaplan)',
-    'guido@google.com (Guido van Rossum)',
-)
-
```

Comment:
neither have actual commits in the repo. Looking into the history, it's likely that the very first version of this file was copied from elsewhere in Google and guido wrote that file.